src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -62,7 +63,7 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
             if (mouseWheelLegacyEvent.wheelDelta) {
                 wheelDelta = mouseWheelLegacyEvent.wheelDelta;
             } else {
-                wheelDelta = -(event.deltaY || event.detail) * 60;
+                wheelDelta = -(event.deltaY || event.detail);

```

Comment:
This was causing a wheelDelta change of 6000 as opposed to the original 100.  This is because we were just pulling the value for wheelDelta (any change, as opposed to a change on a specific axis).  This leads me to believe that this was outdated and never got triggered so we didn't know there was a problem.